From e5a8ecb1687fd8eb84080b84350dd7517d722c23 Mon Sep 17 00:00:00 2001
From: Michael Tautschnig <mt@debian.org>
Date: Thu, 1 Jun 2017 06:31:24 +0000
Subject: [PATCH 2/2] Make type of event_session_id consistent

libtool: link: gcc -D_GNU_SOURCE -Wno-pointer-sign -g -O0 -fdebug-prefix-map=/home/admin/workspace/sid-goto-cc-audit/audit-2.6.7=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z -Wl,relro -Wl,-z -Wl,now -Wl,--as-needed -o .libs/aureport aureport.o auditd-config.o ausearch-llist.o aureport-options.o ausearch-string.o ausearch-parse.o aureport-scan.o aureport-output.o ausearch-lookup.o ausearch-int.o ausearch-time.o ausearch-nvpair.o ausearch-avc.o ausearch-lol.o  -L../lib /home/admin/workspace/sid-goto-cc-audit/audit-2.6.7/debian/build/lib/.libs/libaudit.so -L../auparse /home/admin/workspace/sid-goto-cc-audit/audit-2.6.7/debian/build/auparse/.libs/libauparse.so
../../../src/ausearch-options.h:42:1: error: error: conflicting types for variable `event_session_id'
old definition in module `aureport-options' file ../../../src/aureport-options.c line 62
signed int
new definition in module `ausearch-parse' file ../../../src/ausearch-options.h line 42
uint32_t
Makefile:562: recipe for target 'aureport' failed
make[5]: *** [aureport] Error 1
---
 src/aureport-options.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/aureport-options.c b/src/aureport-options.c
index 9a0fc18..e9a90bf 100644
--- a/src/aureport-options.c
+++ b/src/aureport-options.c
@@ -59,7 +59,8 @@ const char *event_uuid = NULL;
 const char *event_vmname = NULL;
 long long event_exit = 0;
 int event_exit_is_set = 0;
-int event_ppid = -1, event_session_id = -2;
+int event_ppid = -1;
+uint32_t event_session_id = -2;
 int event_debug = 0, event_machine = -1;
 
 /* These are used by aureport */
-- 
2.1.4

